base_station/gui/src/store/modules/autonomy.js

```diff
@@ -9,8 +9,7 @@ const state = {
   playbackLat: [],
   playbackLon: [],
   playbackBearing: [],
```

Comment:
can you add a distinction of which type of bearing this is? like is this the rover's bearing (from the imu?) so it's clear how its different from gpsBearing

Reply:
Anything that isn't labeled refers to the rover odometry, but I'll add that